ui/src/app/views/export-data-set-modal.tsx

```diff
@@ -132,18 +162,32 @@ class ExportDataSetModal extends React.Component<
       }
     });
     return <Modal loading={loading || notebooksLoading}>
-      <ModalTitle>Export {dataSet.name} to Python Notebook</ModalTitle>
+      <ModalTitle>Export {dataSet.name} to Notebook</ModalTitle>
       <ModalBody>
         <Button data-test-id='code-preview-button'
                 onClick={() => this.setState({seePreview: !seePreview})}>
           {seePreview ? 'Hide Preview' : 'See Code Preview'}
         </Button>
         {seePreview && <React.Fragment>
-          {queries.length === 0 && <SpinnerOverlay />}
-          <TextArea disabled={true} onChange={() => {}} style={{marginTop: '1rem'}}
+          {Array.from(queries.values())
+            .filter(query => query !== undefined).length === 0 && <SpinnerOverlay />}
+          <div style={styles.codePreviewSelector}>
+            <TabButton onClick={() => this.setState({previewedKernelType: KernelTypeEnum.Python})}
```

Comment:
Can these buttons be refactored to take a generic `KernelTypeEnum` ?

Pseudocode:
```
KernelTypeEnum.foreach { kernelType ->
<TabButton onClick={() => this.setState({previewedKernelType: kernelType})}
                       active={previewedKernelType === kernelType}
                       style={styles.codePreviewSelectorTab}
                       disabled={queries.get(kernelType) === undefined}>
  kernelType.ToString()
</TabButton>
}
```